examples/dummy/asyncapi.yaml

```diff
@@ -35,24 +42,36 @@ channels:
           properties:
             my-custom-header:
               type: string
-              const: 'custom value'
-
-    publish:
-      operationId: onUserSignedUp
-      message:
-        $ref: '#/components/messages/UserSignedUp'
-      bindings:
-        mqtt:
-          qos: 2
-          retain: true
-          bindingVersion: 0.1.0
-    subscribe:
-      message:
-        $ref: '#/components/messages/UserSignedUp'  
+              const: custom value
   server/announce:
-    subscribe:
-      message:
+    address: server/announce
+    messages:
+      subscribe.message:
```

Comment:
Watch out with the word `subscribe`. We should avoid it at all costs to not create more confusion.